logos/logos-ui/src/app/features/passkeys/passkeys.scss

```
@@ -0,0 +1,93 @@
+}
+
+// ── Passkey list ──────────────────────────────────────────────────────
+.passkey-list {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.passkey-row {
+  @include glass-card;
+  padding: 14px 18px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)